code/game/objects/items/devices/radio/headset.dm

```diff
@@ -253,14 +255,16 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 
 /obj/item/radio/headset/heads/hos
```

Comment:
So I know you mentioned that helmets offer ear protection as part of the justification to moving away from making security bowman's the default headset, but helmets do not have the ear protection component. This should likely be handled before going forward.

Reply:
I just reverted that proposal, will do the second John proposal